### PR TITLE
Update ufs-weather-model 

### LINKED ISSUE #2490: Update ufs-weather-model hash in g-w
### What new functionality do you need?

Update of ufs-weather-model with recent bug-fixes for running C768 on hera. 

### What are the requirements for the new functionality?

Update ufs-weather-model hash

### Acceptance Criteria

Updated ufs-weather-model hash 

### Suggest a solution (optional)

Adding this issue here to see if anyone already had a PR in the works as to not duplicate efforts.

### PR DESCRIPTION
**Pull request by JessicaMeixner-NOAA**
# Description
Updates UFS model to the commit from today.  A commit is coming soon that will update RDHPCS to 1.6.1, but i thought I'd open this now.  This should resolve the issue and allow C768 runs on Hera and allow for CICE to run on WCOSS2 (due to library updates to allow linking). 

From what I can tell, all updates needed were done by @HenryWinterbottom-NOAA which were updates for CICE 

Fixes #2490

# Type of change

- Maintenance: Routine update of model 

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
In Progress... 

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
Looks good.
Thanks to @HenryWinterbottom-NOAA for the updates needed in `parsing_namelists_CICE.sh`

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Conditionally approved

**Review by aerorahul — CHANGES_REQUESTED** [review]
The UFS-weather-model update is fine.
I just noticed the yamls are in the `weekly` directory.
We can't run these resolutions weekly on all platforms.
These exist in the `hires` and should be left there.

**Comment by WalterKolczynski-NOAA** [comment]
Wave post points is timing out. Is there a change in the model that might be making that run more slowly? Otherwise, @DavidHuber-NOAA recently had a PR (#2588) that set a cap on the number of PEs/node at 40 to solve Hercules IO issues with the job; perhaps that should be lower? Or maybe the PEs aren't being spread to use all nodes?

**Comment by JessicaMeixner-NOAA** [comment]
> Wave post points is timing out. Is there a change in the model that might be making that run more slowly? Otherwise, @DavidHuber-NOAA recently had a PR (#2588) that set a cap on the number of PEs/node at 40 to solve Hercules IO issues with the job; perhaps that should be lower? Or maybe the PEs aren't being spread to use all nodes?

@WalterKolczynski-NOAA the code for the wave point post job did not change, so my guess is we either got slow nodes (not sure if this is a hercules and orion issue or just an orion issue as I haven't used hercules enough) or one of the other issues you are describing.  I do know there will soon be a module udpate to move to 1.6 spack on hercules (and other RDHPCS) not sure if that will help or hurt this issue.

**Comment by DavidHuber-NOAA** [comment]
@WalterKolczynski-NOAA @JessicaMeixner-NOAA I have made a few optimizations to the wavepostpnt jobs this morning and will try them out today.  I'll let you know if there is an improvement to speed.  You can check them out [here](https://github.com/DavidHuber-NOAA/global-workflow/commit/d87a550fb2a635221744db453d6d880e92523b70).

**Comment by JessicaMeixner-NOAA** [comment]
@DavidHuber-NOAA let me know if there's anything I can do to help with the point output job debugging on hercules issues.

**Comment by aerorahul** [comment]
@JessicaMeixner-NOAA 
If you can move the yamls from `ci/cases/weekly/` into `ci/cases/hires/`, we would like to merge this PR.
A patch for Hercules failure will be addressed in a separate PR as the failure is not related to the changes in this PR.
Thanks!

**Comment by JessicaMeixner-NOAA** [comment]
@aerorahul - can you double check what i did? I think I did as asked.

**Comment by aerorahul** [comment]
> @aerorahul - can you double check what i did? I think I did as asked.

We would like to keep:
```
C384_S2SWA.yaml
C384_atm3DVar.yaml
C384C192_hybatmda.yaml
```
in the `ci/cases/weekly/` directory.  
Please move them back from `ci/cases/hires/` to `ci/cases/weekly/`. 

I was referring to:
```
C1152_S2SW.yaml
C768_S2SW.yaml
```
to be moved to `ci/cases/hires`.
They have been removed in the last commit https://github.com/NOAA-EMC/global-workflow/pull/2646/commits/77b4ff03408fd4b3a00c5f9cae14bc397d33888e

File: ci/cases/hires/C384C192_hybatmda.yaml
Comment (aerorahul) [inline ci/cases/hires/C384C192_hybatmda.yaml:1]: please undo this move

File: ci/cases/hires/C384_S2SWA.yaml
Comment (aerorahul) [inline ci/cases/hires/C384_S2SWA.yaml:1]: please undo this move

File: ci/cases/hires/C384_atm3DVar.yaml
Comment (aerorahul) [inline ci/cases/hires/C384_atm3DVar.yaml:1]: please undo this move

**Review by aerorahul — CHANGES_REQUESTED** [review]
please undo the move of the weekly yamls

**Comment by JessicaMeixner-NOAA** [comment]
Thanks for clarifying what you wanted, I hadn't realized I had moved any yamls in this PR so I was a bit confused.   Now I think this is what you wanted to see. Please let me know if I am not correct.

**Review by aerorahul — APPROVED** [review]
Looks good.
Thanks for addressing the review comments.

**Pull request merged**